dom.bs

```diff
@@ -4363,6 +4363,26 @@ dom-Range-extractContents, dom-Range-cloneContents -->
  <a for=tree>children</a> of <var>node</var> and append them to <var>copy</var>, with
  <var>document</var> as specified and the <i>clone children flag</i> being set.
 
+ <li>
+  <p>If <var>node</var> is a <a for=Element>shadow host</a> whose <a for=/>shadow root</a>'s
+  <a for=ShadowRoot>clonable</a> is true:
+
+  <ol>
+   <li><p>Run <a>attach a shadow root</a> with <var>copy</var>, <var>node</var>'s <a for=/>shadow
```

Comment:
No wrapping in phrasing-level elements such as `<a>`. (Unfortunately the requirements for DOM and HTML are still different.)

Reply:
It makes inline search harder.

Reply:
But if you're working on WHATWG/html you're already used to searching for "my\s+multi\s+word\s+concept", right?